BugFablesLib/Sources/Names/Medals.cs

```diff
@@ -77,6 +77,7 @@ public static partial class BfVanillaNames
     "Miracle Matter",
     "Victory Buzz",
     "Triumph Buzz",
+    "Ambusher",
```

Comment:
This doesn't match the game: this enum needs to match the one that's in game so the Ids matches, but all the new 1.2.0 medals were added at the end of the enum and in a specific order.

I also find it inefficient to add only a single one when you could add all of them, but you HAVE to follow the internal ids which are listed here: https://aldelaro5.github.io/Bug-Fables-Internal-Docs/Enums%20and%20IDs/Medal/#medals-table

Currently, you made this entry the new "crazy prepared" and made every other ids below off by one.